R/readTechCosts.R

```diff
@@ -59,29 +79,470 @@ readTechCosts <- function(subtype = "PowerAndHeat") { # nolint
 
 
   if (subtype == "PowerAndHeat") {
-    x <- read.csv("power_and_heat_energyf.csv")
-    names(x) <- c("tech", "value", "year", "variable", "unit")
-    x <- filter(x, x[["tech"]] != "Refurbishment of existing nuclear reactors")
-    x[["value"]] <- as.numeric(x[["value"]])
-    x <- as.quitte(x)
+
+      df <- read_excel("REF2020_Technology Assumptions_Energy.xlsx", sheet = "Power&Heat", range = "A2:V80")
+
+      df <- df[,-c(14:21)]
+      df2 <- df
+      df <- df[-c(1,2),]
+
+      df[,2] <- as.numeric(unlist(df[,2]))
+      df[,6] <- as.numeric(unlist(df[,6]))
+      df[,10] <- as.numeric(unlist(df[,10]))
+      df[,14] <- as.numeric(unlist(df[,14]))
+
+      dfp <- pivot_longer(df, cols = c(2:14))
+
+      dfp[339:342,3] <- (1500+850)/2
+
+      dfp$unit <- NA
+      dfp[["unit"]] <- "EUR/kW"

```

Comment:
it would be better not to have any hard-coded data. suggested change: read the values from the data

Reply:
ok